.changeset/cyan-weeks-vanish.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cloudoperators/juno-app-supernova": patch
```

Comment:
please turn this into a minor change. It's a new feature. Patch is only for bug fixes or small changes that don't introduce a new feature/behavior
```suggestion
"@cloudoperators/juno-app-supernova": minor
```